sections/06_discussion.md

```diff
@@ -198,6 +198,124 @@ influence healthcare decisions? Or, is deep learning a hypothesis generation
+representations from single data type. These individual representations are
+further concatenated before or within fully-connected layers. FIDDLE
+[@tag:Eser2016_fiddle] is an example of multimodal CNNs that represents an
+ensemble of individual networks that take as inputs a number of genomic
+datasets, including NET-seq, MNase-seq, ChIP-seq, RNA-seq, and
+raw DNA sequence to predict Transcription Start Site-seq (TSS-seq) outputs.
+The combined model radically improves performance over separately trained
+datatype-specific networks, suggesting that it learns the synergistic
+relationship between datasets.
+
+Despite demonstrated improvements, transfer learning approaches also pose
+a number of challenges. As mentioned above, there are no theoretically sound
+principles for pre-training and fine-tuning. Most best practice recommendations
+are heuristic and have to take into account additional hyper-parameters that
+depend on specific deep architectures, sizes of pre-training and target
+datasets, and similarity of domains. Unfortunately, not many negative results
+are presented in study publications. As rare example, DeepChem, a framework
+that successfully improved results for low-data drug discovery by implementing
+one-shot learning techniques [@tag:AltaeTran2016_one_shot],
+was not able to achieve any predictive power with transfer learning on
+closely related testing tasks from different datasets.
+This long shot-term memory (LSTM) based method addressed training set
+size limitations by implementing one-shot learning approaches that aim to
+learn information about objects from few training examples. DeepChem showed
+significant improvement in predictive power for a variety of problems meaningful
+for low-data drug discovery and in its ability to generalize new experimental
+assays, related but not identical to the training collection. Originally it was
+used separately on the Tox21 dataset, containing 12 nuclear receptor assays
+related to human toxicity, and the SIDER dataset containing information on
+marketed medicines and their observed adverse reactions. For both prediction
+tasks, DeepChem demonstrated superior performance in low-data regime.
+However, when trained on Tox21 data and tested on SIDER, the model failed,
+clearly demonstrating a limit to the cross-task generalization capability
+of one-shot models.
+
+Overall, multimodal and transfer learning strategies demonstrate high potential
+for many biomedical applications that are otherwise limited by data volume.
+These methods not only inherit most of methodological issues from natural image,
+text, and audio domains, but also pose new challenges, specific to biological
+data. We suggest that this class of models deserve deeper investigation to
+establish best practices and determine limits for the transferability of
+features between various close related and distant learning tasks. Making
+negative results and source code public helps to accelerate progress in this
```

Comment:
Data sharing likely plays an important role as well, right?  What about pre-trained models as a way to jump start transfer learning?